blazor/rich-text-editor/tools/dialogSettings.md

```diff
@@ -0,0 +1,33 @@
+---
+layout: post
+title: Customization of Blazor Rich Text Editor Dialog | Syncfusion
+description: Check out and learn here all about DialogSettings in Syncfusion Blazor RichTextEditor component and more.
```

Comment:
description: Check out and learn about DialogSettings in Syncfusion Blazor RichTextEditor component and more.